env_batch.xml

```diff
@@ -103,7 +103,7 @@
       <valid_values/>
       <desc>The machine wallclock setting.  Default determined in config_machines.xml can be overwritten by testing</desc>
     </entry>
-    <entry id="JOB_QUEUE" value="economy">
+    <entry id="JOB_QUEUE" value="premium">
```

Comment:
Are we using the premium queue because there's time pressure to replace the missing files?